uploadNeuroDB/tarchiveLoader

```diff
@@ -183,10 +189,9 @@ unless ($tarchive =~ m/$tarchivePath/i) {
 }
 
 unless (-e $tarchive) {
-    print "\nERROR: Could not find archive $tarchive. \n ".
-          "Please, make sure the path to the archive is correct. ". 
-          "Upload will exit now.\n\n\n";
-    exit 4;
+    print STDERR "\nERROR: Could not find archive $tarchive.\n"
+                 . "Please, make sure the path to the archive is correct.\n\n";
+    exit $NeuroDB::ExitCodes::ARG_FILE_DOES_NOT_EXIST;
```

Comment:
@cmadjar 
you forgot ONE exit code on success to change it :) (yes only ONE)
please scroll to line 251 (249 in original script before your changes)... I can not comment there so I am putting it here instead

Reply:
Good catch!!